Cargo.toml

```diff
@@ -46,6 +46,10 @@ zeroize = { version = "1.8", features = ["zeroize_derive"], default-features = f
 # error handling
 thiserror = { version = "1.0", default-features = false }
 anyhow = { version = "1.0", default-features = false }
+snarkvm = { version = "1.1.0" } # TODO: include only the features we need
+# snarkvm = { version = "1.1.0", default-features =  false, features = ["parameters"]}
```

Comment:
What's the context for this commented out entry?

Reply:
Also, it seems this toml file has sections delimited by comments, and our new entries are right under `# error handling`. Can we follow the original structure and use/create a dedicated section too?